docs/Contributors_Guide/noaa_install.rst

```diff
@@ -0,0 +1,1093 @@
+     - **Account Access**
+   * - Ursa
+     - /contrib/met/modulefiles/met
+     - role.metplus
+   * - Hera
+     - /contrib/met/modulefiles/met
+     - role.metplus
+   * - Jet
+     - /contrib/met/modulefiles/met
+     - role.metplus
+   * - Gaea
+     - /usw/met/modulefiles/met
+     - personal
+   * - Orion
+     - /apps/contrib/modulefiles/met
+     - role-ovp
+   * - Hercules
+     - /apps/contrib/modulefiles/met
+     - role-ovp
+
+Download the existing installation modulefile for the appropriate system.
+These modulefiles are located in the
+`MET GitHub repository <https://github.com/dtcenter/MET>`_
+in the **internal/scripts/installation/modulefiles** directory and are named with the format
+*<X.Y.Z>_<system-name>*. For example, *12.1.0_jet* or *12.1.0_ursa*.
+
+To download the file for **Ursa** for MET version X.Y.Z, for example, run:
+
+.. code-block::
+
+   wget https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_vX.Y/internal/scripts/installation/modulefiles/X.Y.Z_ursa
+
+.. note::
+
+   The :code:`wget` command above will get the modulefile for the
+   MET X.Y.Z release. Replace the *main_vX.Y* with the actual version numbers or
+   with *develop* for a betaN or rcN release, and replace
+   the *X.Y.Z* in the *X.Y.Z_<system-name>* filename with the actual version numbers.
+
+.. warning::
+
+   Note that the link above links to the **RAW** content of the file. It is essential to
+   download the raw format, otherwise the file will contain unwanted HTML information
+   and will not work appropriately. 
+
+If installing an official release, rename the file simply X.Y.Z. For example:
+
+.. code-block::
+
+   mv X.Y.Z_ursa X.Y.Z
+
+If installing a beta release, rename the file X.Y.Z-betaN. For example:
+
+.. code-block::
+   
+   mv X.Y.Z_ursa X.Y.Z-betaN
+
+Open the file using the editor of your choice and change any references to
+X.Y.Z to X.Y.Z-betaN. Save the file.
+   
+If installing a rc release, rename the file X.Y.Z-rcN. For example:
+
+.. code-block::
+
+   mv X.Y.Z_ursa X.Y.Z-rcN
+
+Open the file using the	editor of your choice and change any references	to
+X.Y.Z to X.Y.Z-rcN. Save the file.
+
+**Review the file to ensure no other updates need to be made.**
+
+
+Installing METplus
+==================
+
+Sample Data
+-----------
+
+The METplus Sample Data is available online at this
+`location <https://dtcenter.ucar.edu/dfiles/code/METplus/METplus_Data/>`_.
+Click on the link for the desired version of METplus to access the data for that version.
+
+Use the following commands to download the desired data, replacing the X.Y with the
+appropriate version number:
+
+.. code-block::
+
+   wget -r --no-parent https://dtcenter.ucar.edu/dfiles/code/METplus/METplus_Data/vX.Y/met_test/
+   wget -r --no-parent https://dtcenter.ucar.edu/dfiles/code/METplus/METplus_Data/vX.Y/model_applications/
+
+.. note::
+
+   Due to the size of the data, the wget commands could potentially take hours to download all of the data.
+
+
```

Comment:
Sorry I didn't mention this earlier, but after the wget commands the users will have to do the following to move the data to the correct directory, as wget hides the needed directories in a chain of subdirectories:
```
mv dtcenter.ucar.edu/dfiles/code/METplus/METplus_Data/vX.Y/met_test .
mv dtcenter.ucar.edu/dfiles/code/METplus/METplus_Data/vX.Y/model_applications .
rm -rf dtcenter.ucar.edu
```

Reply:
Oh! I'm so glad you mentioned this.  I did not notice. Thank you so much! 
Due to the reference to the current directory (`.`) in the `mv` commands, I decided to combine the Sample Data and the Data Installation Location sections to be more clear about where the user should be when downloading the data. I just pushed it up, so the RTD build may take about five minutes still.  Could you please re-review the wording of the new Sample Data Installation Location? And then approve the PR if you think it looks ok? Please don't hesitate to add any new suggestions for improvement. Thanks for all of your help.